src/search/utils/component_errors.h

```diff
@@ -0,0 +1,28 @@
+#ifndef UTILS_COMPONENT_ERRORS_H
+#define UTILS_COMPONENT_ERRORS_H
+
+#include "exceptions.h"
```

Comment:
Is this include needed?

Reply:
Looks like it is used in line 9 for the base class.